### ISSUE TITLE
exploit template script missing pwnup.mako

### ISSUE THREAD

**Issue by Caesurus** [Gallopsled/pwntools#977]
I was trying to use the new template functionality.

My system is Ubuntu 16.04 (sorry did not have a 14.04 system available to test with). I made sure to update to the latest versions using `pip install --upgrade pwntools`

When I run the command: `$ pwn template`
I get the following:
```
Traceback (most recent call last):
  File "/usr/local/bin/pwn", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/pwnlib/commandline/main.py", line 51, in main
    commands[args.command](args)
  File "/usr/local/lib/python2.7/dist-packages/pwnlib/commandline/template.py", line 46, in main
    template = lookup.get_template('pwnup.mako')
  File "/usr/local/lib/python2.7/dist-packages/mako/lookup.py", line 261, in get_template
    "Cant locate template for uri %r" % uri)
mako.exceptions.TopLevelLookupException: Cant locate template for uri 'pwnup.mako'
```

**Comment by TethysSvensson** [comment]
This should be fixed in 3.6.1. Please reopen of that is not the case.

**Comment by Caesurus** [comment]
Verified as fixed on my system with the update to 3.6.1. Thanks
